saritasa_invocations/poetry.py

```diff
@@ -0,0 +1,47 @@
+import invoke
+
+from . import printing
+
+
+@invoke.task
+def install(context: invoke.Context) -> None:
+    """Install dependencies via poetry."""
+    printing.print_success("Install dependencies with poetry")
+    context.run("poetry install --sync")
+
+
+@invoke.task
+def update(
+    context: invoke.Context,
+    groups: str = "",
+    params: str = "",
+):
+    """Update dependencies with respect to version constraints.
+
+    https://python-poetry.org/docs/dependency-specification/
+
+    """
+    printing.print_success("Update dependencies")
+    context.run(f"poetry up {params} {_parse_groups(groups)}")
+
+
+@invoke.task
+def update_to_latest(
+    context: invoke.Context,
+    groups: str = "",
+    params: str = "",
+):
+    """Update dependencies to latest versions."""
+    printing.print_success("Update dependencies to latest versions")
+    context.run(f"poetry up --latest {params} {_parse_groups(groups)}")
```

Comment:
Is this implementation uses `poetry-plugin-up`?
If yes:
* do we need to add it as (optional) dependency?
* describe this in docstring

Reply:
Updated docs, `poetry-plugin-up` is a plugin for poetry, It has different installation methods and shouldn't be a requirement